ide/Espressif/ESP-IDF/setup_win.bat

```diff
@@ -0,0 +1,415 @@
+if EXIST config_h_%FileStamp%.bak (
+    echo;
+    echo Found prior config.h in  %SCRIPTDIR%\config_h_%FileStamp%.bak
+    echo;
+    dir config_h_%FileStamp%.bak | findstr config_h_%FileStamp%.bak
+    echo;
+
+    :: clear any prior errorlevel
+    call;
+    choice /c YN /m "Use prior config.h  in  %WOLFSSHLIB_TRG_DIR%\include\ "
+    if errorlevel 2 GOTO :NO_CONFIG_RESTORE
+
+    echo new config                                            >  %WOLFSSHLIB_TRG_DIR%\include\config.h
+    call;
+    xcopy config_h_%FileStamp%.bak                                %WOLFSSHLIB_TRG_DIR%\include\config.h /Y
+    if %errorlevel% NEQ 0 SET COPYERROR=true
+
+) else (
+    echo;
+    echo Prior config.h not found. Using default file.
+    echo;
+
+    echo new config                                            > %WOLFSSHLIB_TRG_DIR%\include\config.h
+    call;
+    xcopy  %WOLFSSH_ESPIDFDIR%\dummy_config_h.                   %WOLFSSHLIB_TRG_DIR%\include\config.h             /F /Y
+    if %errorlevel% NEQ 0 SET COPYERROR=true
+)
+::******************************************************************************************************
+:NO_CONFIG_RESTORE
+::******************************************************************************************************
+
+:: Check if operator wants to keep prior config.h
+if EXIST user_settings_h_%FileStamp%.bak (
+    echo;
+    echo Found prior user_settings.h in  %SCRIPTDIR%\user_settings_h_%FileStamp%.bak
+    echo;
+    dir user_settings_h_%FileStamp%.bak | findstr user_settings_h_%FileStamp%.bak
+    echo;
+
+    :: clear any prior errorlevel
+    call;
+    choice /c YN /m "Use prior user_settings.h  in  %WOLFSSHLIB_TRG_DIR%\include\ "
+    if errorlevel 2 GOTO :NO_USER_SETTINGS_RESTORE
+
+    echo;
+    call;
+    xcopy user_settings_h_%FileStamp%.bak    %WOLFSSHLIB_TRG_DIR%\include\user_settings.h /Y
+    if %errorlevel% NEQ 0 SET COPYERROR=true
+
+    :: TODO do we really need to replicate the user_settings.h here?
+    xcopy user_settings_h_%FileStamp%.bak    %WOLFSSHLIB_TRG_DIR%\wolfssl\include\user_settings.h /Y
+    if %errorlevel% NEQ 0 SET COPYERROR=true
+) else (
+    echo;
+    :: TODO do we really need to replicate the user_settings.h here?
+    echo Prior user_settings.h not found.  Using file:  %WOLFSSLLIB_TRG_DIR%\include\user_settings.h
+    echo new file >                                     %WOLFSSHLIB_TRG_DIR%\include\user_settings.h
+    xcopy %WOLFSSLLIB_TRG_DIR%\include\user_settings.h  %WOLFSSHLIB_TRG_DIR%\include\user_settings.h  /Y
+
+    echo;
+)
+
+::******************************************************************************************************
+:NO_USER_SETTINGS_RESTORE
+::******************************************************************************************************
+
+
+echo;
+echo Copying CMakeLists.txt to %WOLFSSHLIB_TRG_DIR%\
+xcopy %WOLFSSH_ESPIDFDIR%\libs\CMakeLists.txt                %WOLFSSHLIB_TRG_DIR%\                             /F

```

Comment:
wolfSSH doesn't have a CMakeLists.txt file.

Reply:
the new script now includes default make files that were missing